### PR TITLE
Beam supports wings

### PR DESCRIPTION
**Pull request by frmanso**
The wings of the beam support on C-side are coded here. They link the support bars of the beam pipe to the main cage. These wings are close to the MFT detector.

### CODE REVIEW

File: Detectors/Passive/src/Pipe.cxx [hunk 1]
```diff
@@ -1509,9 +1637,9 @@ void Pipe::ConstructGeometry()
     dovX -= 0.0625;
     dovY += 0.075;
     char vname[20];
-    snprintf(vname, 20, "voRB242CuOvTransV%d", i);
+    sprintf(vname, "voRB242CuOvTransV%d", i);
```
Comment (mconcas) [inline Detectors/Passive/src/Pipe.cxx]: Any specific reason to do so? `sprintf` is deprecated and should be avoided. (see also CI warnings)

File: Detectors/Passive/src/Pipe.cxx [hunk 2]
```diff
@@ -1509,9 +1637,9 @@ void Pipe::ConstructGeometry()
...
     voRB242CuOvTransV[i] = new TGeoVolume(vname, new TGeoEltu(dovX, dovY, 1.0), kMedCuHC);
-    snprintf(vname, 20, "voRB242CuOvTransI%d", i);
+    sprintf(vname, "voRB242CuOvTransI%d", i);
```
Comment (mconcas) [inline Detectors/Passive/src/Pipe.cxx]: same as  above

**Comment by mconcas** [comment]
Hello, @frmanso thanks for the PR.
I comment here from ITS just to understand if this is going to complement the addition from @mario6829  described in this presentation: https://indico.cern.ch/event/1300636/contributions/5468571/attachments/2674261/4636801/sitta_WP2_230627.pdf
As a confirmation that there is no overlap.

**Comment by frmanso** [comment]
Hello Mario,
    I discover the slides... and all the questions. In fact, I coded these wings in april 2022, after the coding of the support bars. I even gave the full code (new Pipe.cxx by e-mail to him) because I wanted to verify the fixation of these wings to the main cage, Mario was in charge of this cage. I never get any response (Corrado and Mario) and decide to froze the code. These wings are close to the latest MFT disks and will participate to increase the backgound on it, so MFT needs to have this part described.

File: Detectors/Passive/src/Pipe.cxx (hunk 1 above)
Comment (frmanso) [reply Detectors/Passive/src/Pipe.cxx]: Unfortunately, it seems that I didn't work on the latest version of the PIPE code... sorry for that, I should kill this PR and integrate the wings in the current version of PIPE either modify on the fly the code if it's related to few lines.

**Comment by mario6829** [comment]
Dear @frmanso many thanks for your clarifications about questions on my slides. I agree with your implementation of these missing wings.
@mconcas I verified that there is no problem with the current elements in the cage.
Cheer. Mario

**Comment by mconcas** [comment]
> Dear @frmanso many thanks for your clarifications about questions on my slides. I agree with your implementation of these missing wings. @mconcas I verified that there is no problem with the current elements in the cage. Cheer. Mario

Hi @mario6829  thanks for checking!

**Review by mconcas — APPROVED** [review]

**Comment by shahor02** [comment]
Hi, does this mean that the geometries need to be regenerated? I assume that for the barrel material LUT it is not needed (if the extra material is not in the ITS acceptance).

**Comment by shahor02** [comment]
The O2 for data and MC production will be frozen tomorrow, therefore, unless there will be other comments, I'll merge this after 5pm, add it to cherry-picking requests and regenerate geometry (but not mat LUT as the new material is outside of the barrel acceptance).

**Review by sawenzel — APPROVED** [review]

**Comment by amorsch** [comment]
+1

**Comment by shahor02** [comment]
For the record: https://alice.its.cern.ch/jira/browse/O2-4502

**Pull request merged**